cockpit-frontend/src/components/Impediment/Impediment.jsx

```diff
@@ -32,9 +32,13 @@ export default function Impediment(props) {
   const [number, setNumber] = React.useState(0);
   const [selectedImpediment, setSelectedImpediment] = React.useState(0);
   const classes = useStyles();
+  const myRef = useRef();
 
   useEffect(() => {
     setOpen(false);
+    setName('');
```

Comment:
create a function for the initialization of states, they are duplicated for several times in this script